cli/internal/cliparamsatisfier/inputsrc/slice/slice.go

```diff
@@ -14,6 +14,9 @@ func New(
 		// get parts
 		parts := strings.SplitN(arg, sep, 2)
 		inputName := parts[0]
+		if len(parts) < 2 {
```

Comment:
can `parts[0]` npe? If so should we move this check up a line?

Reply:
[Technically yes.](https://go.dev/play/p/KT4esa5e8kp) I don't think it can in real usage, since I don't think it's possible to get it to be an empty string with the way arg parsing works.

But, it definitely makes sense to move this, it won't have a negative impact.